### DISCUSSION TITLE
Suggestion: Show pictures requirements before trying

### DISCUSSION THREAD

**Discussion by Roseghost** [alkem-io/alkemio#482] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/64191883/144750176-20131784-0891-4dc8-a7c7-5c5d12209616.png)
 I got this after uploading a profile picture that fits the given bytes. Perhaps is smarter to give this information beforehand?

**Comment by techsmyth** [comment, marked as answer]
Hi Rosa, 

yes the whole area of image management is in need of a refresh. I have put up an epic for this: https://github.com/alkem-io/alkemio/issues/486

Probably needs to be pretty high up the backlog...feel free to add ideas / requirements directly in there.

Cheers, Neil
